cmd/bucket-quota.go

```diff
@@ -65,10 +65,10 @@ func (sys *BucketQuotaSys) Init(objAPI ObjectLayer) {
 func (sys *BucketQuotaSys) GetBucketUsageInfo(bucket string) (BucketUsageInfo, error) {
 	v, err := sys.bucketStorageCache.Get()
 	if err != nil && v != nil {
-		logger.LogIf(GlobalContext, fmt.Errorf("unable to retrieve usage information for bucket: %s, relying on older value cached in-memory: err(%v)", bucket, err))
+		logger.LogOnceIf(GlobalContext, fmt.Errorf("unable to retrieve usage information for bucket: %s, relying on older value cached in-memory: err(%v)", bucket, err), "bucket-usage-cache"+bucket)
```

Comment:
`"bucket-usage-cache"+bucket` have a bad format. We can add `: `?

Reply:
@jiuker This is the token used to identify duplicates, so it doesn't have to be "nice".

Reply:
@klauspost `-` is ok. But there's no distinction at the moment and I don't think it looks good

Reply:
I have applied the style recommended in the previous PR, however I'll add a slash in between as it looks cleaner 👍

Reply:
Thanks @jiuker, I have added a slash in between 👍